calculate_statistics/settings.py

```diff
@@ -0,0 +1,4 @@
+DATA_DOMAIN = 'https://www.performance.service.gov.uk/data'
```

Comment:
We should probably handle the `DATA_DOMAIN` and `PP_DATASET_TOKEN` in the same way, because the `DATA_DOMAIN` would also be varying across several environments (this script in GOV.UK preview should post to PP preview, etc).